runtime/Makefile

```diff
@@ -52,17 +52,17 @@ configure:
 		-DQIR_STDLIB_INCLUDES=$(QIR_STDLIB_INCLUDES_DIR) \
 		-DENABLE_CODE_COVERAGE=$(CODE_COVERAGE)
 
-$(QIR_STDLIB_INCLUDES_DIR): qir-stdlib
+$(QIR_STDLIB_DIR)/libqir_stdlib.a $(QIR_STDLIB_INCLUDES_DIR)/qir_stdlib.h: qir-stdlib
```

Comment:
I think it makes more sense to add these dependencies in the `qir-stdlib` directory. :)

Reply:
The reason why I prefer is because there are two rules to build the dependencies in two different locations and I think it makes more sense to have the qir build rule in the qir directory. It is not a big issue. If we are going to to remove then your solution is fine. Do you know if this is planned in the QIR stdlib repo?

Reply:
I'm not sure if that's planned for the next version. But, we can definitely suggest it :) I can see your point too. What would you think of simplifying the qir-stdlib/Makefile by removing the second non-phony `target` dependency?